lang/src/org/partiql/lang/util/ExprValueFormatter.kt

```diff
@@ -47,6 +47,8 @@ class ConfigurableExprValueFormatter(private val config: Configuration) : ExprVa
 
                 STRING                                     -> out.append("'${value.scalar.stringValue()}'")
 
+                DATE                                       -> out.append(value.scalar.dateValue().toString())
```

Comment:
I see. In that case the formatted output for the date literals would be the same as the expression. 
`DATE '2012-09-30'` would output `DATE '2012-09-30'` instead of `'2012-09-30'`.
I have created an issue #386 to track this. This requires the `DATE` exprValue comparison to be implemented and is worth a separate PR.